crates/ruff/src/rules/pycodestyle/snapshots/ruff__rules__pycodestyle__tests__E101_E101.py.snap

```diff
@@ -5,8 +5,8 @@ E101.py:11:1: E101 Indentation contains mixed spaces and tabs
    |
 11 | def func_mixed_start_with_tab():
 12 |     # E101
-13 | 	  print("mixed starts with tab")
-   | ^^ E101
+13 |       print("mixed starts with tab")
```

Comment:
Previously, tabs were counted as two characters but rendered as multiple ones. That's why the caret was off.